### ISSUE TITLE
Feature Request: Object Watching

### ISSUE THREAD

**Issue by michaelhart** [marcuswestin/store.js#52]
It'd be really awesome if we could completely drop the need for localStorage.getItem/setItem and store.get/set.

Looking at https://gist.github.com/384583, it looks like it'd be possible to bind an object directly to localStorage. In practice, you could simply associate a few objects with their localStorage keys and then have a field day; no need to worry about get/store ever again. All you'd need to know is that it's an object and it's always up to date.

In other words, this would be unfathomably awesome.

**Comment by marcuswestin** [comment]
Store.js v2.0 has been released with support for storage events and storage value observations: https://github.com/marcuswestin/store.js#user-content-list-of-all-plugins

If you update  you will get this functionality plus a bunch more :)

**Comment by adarshmadrecha** [comment]
@marcuswestin as per #267 This issue is now resolved. I mean, the event does not trigger on another tab.
